src/sentry/models/group.py

```diff
@@ -692,6 +692,13 @@ def get_absolute_url(
                 query = urlencode(params)
             return organization.absolute_url(path, query=query)
 
+    def get_absolute_api_url(self, params: Mapping[str, str] | None = None):
+        query = None
+        path = f"/issues/{self.id}/"
+        if params:
+            query = urlencode(params)
+        return self.organization.absolute_api_url(path=path, query=query)
```

Comment:
do we need to include query params? The published endpoint doesn't look like it takes any? https://docs.sentry.io/api/events/retrieve-an-issue/ :/